scripts/check-coverage.jsh

```diff
@@ -100,6 +100,7 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("sirius-components-task-edit", 3.0),
   new ModuleCoverage("sirius-components-view-diagram-customnodes", 50.0),
   new ModuleCoverage("sirius-components-view-diagram-customnodes-edit", 79.0),
+  new ModuleCoverage("sirius-components-datatree", 100),
```

Comment:
100 for something which is not currently being test is a bit optimistic 😄

Reply:
I set 100 to make sure I didn't forget to write some tests once we agree on what they are